nobiru/nobiru_list.py

```diff
@@ -16,3 +16,10 @@ def delete(self, entity):
             db.session.commit()
         except ValueError:
             abort(404, f"{entity._name()} not found")
+
+    def find(self, id):
+        try:
+            return next(x for x in self if x.json()["id"] == id)
+
+        except StopIteration:
+            abort(404, "Not found")
```

Comment:
`Not found` is a big generic. Is there any way we can specify what was not found?